Makefile

```diff
@@ -18,7 +18,6 @@
 
 SHELL 	:= /bin/bash
 BINDIR	:= bin
-PKG 	:= cloudesf.googleresource.com/gcpproxy
```

Comment:
Note that this is mis-spelled (google**re**source), which is probably why this never worked.